templates/generic/rpi4_64_reference_bb_kirkstone.json

```diff
@@ -0,0 +1,220 @@
+{
+    "platformName": "rpi4_64_reference",
+    "os": "linux",
+    "arch": {
+        "arch": "aarch64",
+        "variant": "v8"
+    },
+    "rdk": {
+        "version": "2024Q3",
+        "supportedFeatures": [
+            "Controller",
+            "DeviceInfo",
+            "Monitor",
+            "OCDM",
+            "TraceControl",
+            "WifiManager",
+            "com.comcast.CoPilot",
+            "com.comcast.DeviceProvisioning",
+            "com.comcast.FrameRate",
+            "com.comcast.HdcpProfile",
```

Comment:
remove all video specific items in all files

Reply:
Addressed Chandra
Please review